src/assets/css/_sass/elements/_buttons.scss

```diff
@@ -5,3 +5,23 @@
   Don't forget hover, active states
 
 */
+
+.button {
+  padding: .575rem .75rem;
+  text-decoration: none;
+  box-shadow: 2px 2px 1px #1a1a1a;
```

Comment:
box shadow mixin - now is a good time to start figuring out the various shadows around the site, and use a single global mixin, or create a few for reuse.
This is also the same as the text-shadows elsewhere